Doc/tutorial/inputoutput.rst

```diff
@@ -279,11 +279,11 @@ Reading and Writing Files
    object: file
 
 :func:`open` returns a :term:`file object`, and is most commonly used with
-two arguments: ``open(filename, mode)``.
+two or three arguments: ``open(filename, mode, encoding=None)``
```

Comment:
We like the encoding to be a keyword for readability so I'd word this similar to "two arguments, often with an encoding keyword when using a text mode" rather than including encoding in the number and mentioning two numbers. it feels more clear to me that way.

Reply:
At this point, we haven't described the "text mode" yet. It is described in below.

How about "two positional arguments and one keyword argument"?
Since binary file is rare than text file, we can focus on text file at this first open() example.